standard/basic-concepts.md

```diff
@@ -253,6 +254,7 @@ The accessibility domain of a nested member `M` declared in a type `T` within
 
 - If the declared accessibility of `M` is `public`, the accessibility domain of `M` is the accessibility domain of `T`.
 - If the declared accessibility of `M` is `protected internal`, let `D` be the union of the program text of `P` and the program text of any type derived from `T`, which is declared outside `P`. The accessibility domain of `M` is the intersection of the accessibility domain of `T` with `D`.
+- If the declared accessibility of `M` is `private protected`, let `D` be the intersection of the program text of `P` and the program text of any type derived from `T`. The accessibility domain of `M` is the intersection of the accessibility domain of `T` with `D`.
```

Comment:
I think the correct specification is:

```suggestion
- If the declared accessibility of `M` is `private protected`, let `D` be the intersection of the program text of `P` and the program text of `T` and any type derived from `T`. The accessibility domain of `M` is the intersection of the accessibility domain of `T` with `D`.
```

Which addresses @KalleOlaviNiemitalo’s observation.

Reply:
Done.